x/emissions/keeper/keeper.go

```diff
@@ -1995,6 +1963,59 @@ func (k Keeper) GetIdsOfActiveTopics(ctx context.Context, pagination *types.Simp
 	}, nil
 }
 
+// Returns active topic ids at block using pagination
+func (k Keeper) GetIdsActiveTopicAtBlock(ctx context.Context, blockHeight int64, pagination *types.SimpleCursorPaginationRequest) ([]TopicId, *types.SimpleCursorPaginationResponse, error) {
```

Comment:
do we really need pagination here?

Reply:
resolved in https://github.com/allora-network/allora-chain/pull/557 + removed GetActiveTopics